src/common/currentSigningMessage.ts

```diff
@@ -0,0 +1,25 @@
+import web3 from "web3";
+
+export default function currentSigningMessage(currentRound: number) {
+  /*
+    Whenever we change the signing message, we should wait until the next round.
+    Determine current round, then create a conditional such that if we change the message in the middle of the round, 
+    it will only apply to the next, to prevent any bugs in revealing.
+    There shouldn't be many reasons for us to change this, this is just to prevent any bugs in the commit -> reveal in a given round.
+  */
+  if (currentRound > SIGNING_MESSAGE_ONE_ROUND_CREATED) {
+    return {
+      message: SIGNING_MESSAGE_ONE,
+      hashedMessage: web3.utils.utf8ToHex(SIGNING_MESSAGE_ONE),
+    };
+  }
+
+  // Default return
+  return {
+    message: SIGNING_MESSAGE_ONE,
+    hashedMessage: web3.utils.utf8ToHex(SIGNING_MESSAGE_ONE),
```

Comment:
Similar question as before -- I don't think this is a hash? Not saying we even need to hash anything! I just don't think we should call this a hash.

For reference, something like `web3.utils.sha3` or `web3.utils.soliditySha3` are hash functions: https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#sha3.